eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -129,7 +128,7 @@ object Helpers {
     }
 
     val localFeeratePerKw = nodeParams.onChainFeeConf.feeEstimator.getFeeratePerKw(target = nodeParams.onChainFeeConf.feeTargets.commitmentBlockTarget)
-    if (isFeeDiffTooHigh(open.feeratePerKw, localFeeratePerKw, nodeParams.onChainFeeConf.maxFeerateMismatch)) throw FeerateTooDifferent(open.temporaryChannelId, localFeeratePerKw, open.feeratePerKw)
+    if (isFeeDiffTooHigh(localFeeratePerKw, open.feeratePerKw, nodeParams.onChainFeeConf.maxFeerateMismatch)) throw FeerateTooDifferent(open.temporaryChannelId, localFeeratePerKw, open.feeratePerKw)
```

Comment:
Why have you inverted `localFeeratePerKw` and `open.feeratePerKw`?

Reply:
Because since the calculation was previously symmetric, it was a mess and `isFeeDiffTooHigh` was called with the wrong order in some places. It didn't matter because the result was the same, but now the result is different, so I had to choose an order.